contracts/Registry.sol

```diff
@@ -244,18 +241,17 @@ contract Registry is IRegistry, ERC165 {
     //  |____/ \___|_|    \_/ |_|\___\___|     |_|  |_|\__, |_| |_| |_|\__|
     //                                                 |___/
 
-    function createServiceRegistration(bytes32 orgName, bytes32 serviceName, bytes32 servicePath, address agentAddress, bytes32[] tags) external {
+    function createServiceRegistration(bytes32 orgName, bytes32 serviceName, bytes metadataURI, bytes32[] tags) external {
 
         requireOrgExistenceConstraint(orgName, true);
         requireAuthorization(orgName, true);
         requireServiceExistenceConstraint(orgName, serviceName, false);
 
```

Comment:
Since metadatauri is critical, is it possible to have a require condition to check for the presence of value:

require(metadataURI.length > 0)